lib/assessment.rb

```diff
@@ -5,35 +5,66 @@
 # Create a class called Dog. It should have, as attributes, name, breed, and age.
 # Make all of those attributes read-only.
 
-##
-# your code here
-##
+class Dog
+  attr_reader :name, :breed, :age
+
+  def initialize(name, breed, age)
+    @name = name
+    @breed = breed
+    @age = age
+  end
+end
+# Comfort: 5
+# Clarity: 5
 
 # Question 2
 # Create a new Dog object.
 
-Answer.dog = nil # replace nil with your code
+spud = Dog.new
```

Comment:
This works. Could you show me you know how to set initial data when you create the new dog? Also, you shouldn't have changed `Answer.dog = `